src/factory.ts

```diff
@@ -34,6 +37,7 @@ export class OperatorFactory {
     suffix: SuffixOperator,
     query: QueryOperator,
     rename: RenameOperator,
+    'fan-out': FanOutOperator,
```

Comment:
nit, but should we just use `fanout` without any special chars?

Reply:
Everywhere else in the code I treat fan out as two words. Do you think it's problematic to use skewer-case?

Reply:
Not problematic; keep it.